commit_files.go

```diff
@@ -39,56 +41,74 @@ func (commitFilesTable) String() string {
 	return printTable(CommitFilesTableName, CommitFilesSchema)
 }
 
-func (commitFilesTable) Resolved() bool { return true }
-
 func (commitFilesTable) Name() string { return CommitFilesTableName }
 
 func (commitFilesTable) Schema() sql.Schema { return CommitFilesSchema }
 
-func (t *commitFilesTable) TransformUp(f sql.TransformNodeFunc) (sql.Node, error) {
-	return f(t)
+func (t *commitFilesTable) WithFilters(filters []sql.Expression) sql.Table {
+	nt := *t
+	nt.filters = filters
+	return &nt
 }
 
-func (t *commitFilesTable) TransformExpressionsUp(f sql.TransformExprFunc) (sql.Node, error) {
-	return t, nil
+func (t *commitFilesTable) WithProjection(colNames []string) sql.Table {
+	return t
```

Comment:
doesn't accept this table pushdowns of projections?

Reply:
same answer